### PR TITLE
[FIZZ-214] Update system activity output directory format

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

File: src/lib/extractor-shared/pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "extractor-shared"
-version = "0.1.9"
+version = "0.1.10"
```
Comment (lalrg) [inline src/lib/extractor-shared/pyproject.toml:3]: We have to be carefull when bumping versions and merging to main

File: src/lib/extractor-shared/extractor_shared/csv_generation/write.py [hunk 2]
```diff
@@ -262,8 +262,11 @@ def write_system_activities(
     output_directory: str
         is the root output directory
     """
+    output_path = os.path.join(output_directory, SYSTEM_ACTIVITY_ROOT_DIRECTORY)
```
Comment (bradbanister) [inline src/lib/extractor-shared/extractor_shared/csv_generation/write.py:265]: FYI, I did this as well for the GC/Canvas shared work.

**Review by bradbanister — APPROVED** [review]
Looks good

**Comment by bradbanister** [comment]
@lalrg I though this was done, but I don't see the IN/OUT tag logic.

**Comment by lalrg** [comment]
> @lalrg I though this was done, but I don't see the IN/OUT tag logic.

Hmm, it is in the extract facade.
image
This is the output that I get
image

**Comment by bradbanister** [comment]
Confirmed everything is correct.  I was looking at an old extractor run. 🤣

**Pull request merged**